meetings/2023/2023-04-25.md

```diff
@@ -0,0 +1,88 @@
+# OpenJS Foundation Cross Project Council Meeting 2023-04-25
+
+## Links
+
+* **Recording**: https://www.youtube.com/watch?v=qQFW9k6MfNc
+* **GitHub Issue**: https://github.com/openjs-foundation/cross-project-council/issues/1058
+
+## Present
+
+* Joe Sepi (@joesepi)
+* Abby Cabunoc Mayes (@abbycabs)
+* Ben Sternthal (@bensternthal)
+* Ruy Adorno (@ruyadorno)
+* Yagiz Nizipli (@anonrig)
+* Tobie Langel (@tobie)
+* Christian Bromann <@christian-bromann)
+* Michael Dawson (@mhdawson)
+* Tierney Cyren (@bnb)
+
+## Agenda
+
+### Announcements
+
+* Node.js v20 released very recently. Huzzah!
+* We want your feedback! Help shape the next 10 years of Node.js. Take the survey: https://linuxfoundation.surveymonkey.com/r/XJ35LYF
+
+### Board Meeting Updates
+
+* No board updates
+
+*Extracted from **cross-project-council-agenda** labeled issues and pull requests from the **openjs-foundation org** prior to the meeting.
+
+### openjs-foundation/cross-project-council
+
+* Simplify CPC Board representation [#1055](https://github.com/openjs-foundation/cross-project-council/pull/1055)
+  * discussing opening a third seat
+  * bylaws make having a board seat for community difficult
+  * discussed having board seats per project (1 seat per 10 projects)
+  * long discussion. We have consensus in meeting. Need to take suggestion to board
+
+* Open Up Community Board Seat [#1054](https://github.com/openjs-foundation/cross-project-council/issues/1054)
+  * see above
+
+* Should we have a co-chair? [#1043](https://github.com/openjs-foundation/cross-project-council/issues/1043)
+  * Joe: I would love help
+
+* CPC Elections: CPC Director & CPC Chairperson  [#1041](https://github.com/openjs-foundation/cross-project-council/issues/1041)
+  * closed; congrats to winners
+
+* Specify how/where to identify individuals from a project to join the CPC [#1026](https://github.com/openjs-foundation/cross-project-council/issues/1026)
+  * Joe and Ben will work on this; Ben is already cataloging lead maintainers and project status
+
+* Enabling Payments for GitHub Codespaces for Projects [#1009](https://github.com/openjs-foundation/cross-project-council/issues/1009)
+  * Ben will monitor costs and usage and provide updates. Closing issue
+
+* Create some documentation / web page to describe the collaboration model of OpenJS projects [#1003](https://github.com/openjs-foundation/cross-project-council/issues/1003)
+  * no update
+
+* Update Contributor Covenant Code of Conduct to v2.1 [#851](https://github.com/openjs-foundation/cross-project-council/pull/851)
+  * we can close this PR once patching work is complete, which it nearly is
```

Comment:
We need to open another pull request to revert some things in the `.patch` file.